css/main.css

```
@@ -1 +1,59 @@
-@import "tailwindcss";
\ No newline at end of file
+@import "tailwindcss";
+@import url("https://fonts.googleapis.com/css2?family=Syne:wght@400..800&display=swap");
+
+html {
+  width: 100%;
+  height: auto;
+  font-family: "Syne" !important;
+}
+@layer components {
+  .navbar-container {
+    max-width: 100%;
```

please use tailwind css classes. please have the example below on how to use tailwind classes in a css file without decalring a normal css style


```suggestion
  .navbar-container {
    @apply w-full list-none flex items-center m-0 p-4 overflow-hidden;
  }
```